test/unittest/tests/wrappers/mock_session.h

```diff
@@ -37,6 +65,8 @@ class MockSession {
 
     // This class is implemented such that it owns, and is responsible for freeing, this pointer
     WT_SESSION_IMPL *_sessionImpl;
+    handler_wrap _handler_wrap;
+    std::vector<std::string> _messages;
```

Comment:
I suggest using `std::list` here, as append operations will generally be more efficient compared to a `std::vector`. `std::vector` is great when you need array semantics.

Reply:
281164070